docs/porting-guides/6.x.x.md

```diff
@@ -1103,22 +1107,104 @@ def get_device_test_catalog(
 ) -> AntaCatalog
 ```
 
-All code using this function must be updated as shown in the following example:
+**Action Required:** Update your code to convert structured configurations into `AvdFabricData` before calling the function.
```

Comment:
we may need to sync on this so that we are consistent in the portigng guide entries for these signature changes